csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests/runtest-docker-gpu.sh

```diff
@@ -27,6 +27,6 @@ docker run --gpus all --rm \
         -e "PackageName=$PackageName" \
         -e "RunTestCsharp=$RunTestCsharp" \
         -e "RunTestNative=$RunTestNative" \
-        onnxruntimeregistry.azurecr.io/internal/azureml/onnxruntimegpubuild:ch35 \
+        onnxruntimeregistry.azurecr.io/internal/azureml/onnxruntimecentosgpubuild:ch5h \
```

Comment:
just curious, what's the naming convention for the image tags?

Reply:
| name | source              |   |   |   |
|--------------------------------------------------|-----------------------------------|---|---|---|
| onnxruntimecpubuild  |              Dockerfile.manylinux2014_cpu                     |   |   |   |
| onnxruntimecentoscpubuild                        | Dockerfile.centos                 |   |   |   |
| onnxruntimecentosgpubuild                        | Dockerfile.centos_gpu             |   |   |   |
| onnxruntimegpubuild                              | Dockerfile.manylinux2014_cuda10_2 |   |   |   |

Reply:
oh, i was asking about the part like "ch5h". is there a way to link these to a specific commit?

Reply:
Oh, the name was auto-generated.  I don't know much on that.